.gitignore

```diff
@@ -12,3 +12,7 @@ Cargo.lock
 .vscode
 .idea
 _tools
+
+cpp/kvproto/*.h
+cpp/kvproto/*.cc
```

Comment:
Why ignore cpp code?

Reply:
@flowbehappy I have the same question

Reply:
Because the cpp code generated by protobuf is likely unportable with other platform.